apps/portals/elportal/src/pages/HomePageV2.tsx

```diff
@@ -65,30 +38,14 @@ export default function HomePage() {
       {/*  />*/}
       {/*</SectionLayout>*/}
       {/*</div>*/}
-      <SectionLayout
-        title={"What's New"}
-        centerTitle
-        ContainerProps={{ className: 'home-spacer' }}
-      >
-        <RssFeedCards
-          url="https://news.eliteportal.org"
-          itemsToShow={3}
-          allowCategories={[]}
-          // mailChimpListName='AMP-AD quarterly newsletter',
-          // mailChimpUrl='https://sagebase.us7.list-manage.com/subscribe/post?u=b146de537186191a9d2110f3a&amp;id=96b614587a',
-          filter={{
-            value: 'whats-new',
-          }}
-        />
-      </SectionLayout>
-      <SectionLayout ContainerProps={{ className: 'home-spacer' }}>
+      <Box sx={{ backgroundColor: 'grey.100' }}>
         <RecentPublicationsGrid
-          sql={topPublicationsSql}
+          sqlString={topPublicationsSql}
           buttonLink="/Explore/Publications"
           buttonLinkText="See More Publications"
           summaryText="Never miss a new finding from the frontiers of aging research."
         />
-      </SectionLayout>
+      </Box>
       {/*<div className={'home-bg-dark'}>*/}
```

Comment:
go ahead and remove this commented code too if you're removing unused components